### PR TITLE
Resolve failing tests 

### PR DESCRIPTION
**Pull request by minouHub**
This issue resolves the failing tests introduced by #633

### CODE REVIEW

**Comment by erwanp** [comment]
still failing but less errors https://app.travis-ci.com/github/radis/radis/jobs/620260402#L1536 

You're missing 

> xvfb-run -a pytest -m "fast and not needs_db_CDSD_HITEMP_PCN and not needs_db_CDSD_HITEMP and **not CDSD-HITEMP-PC**" --cov=./ --durations=10

**Comment by erwanp** [comment]
You should merge anyway then !

**Comment by minouHub** [comment]
omg ... now that hitran.org is not online, pytest is doing weird stuff ... I don't get it, pytest was working fine 6h ago!

**Comment by minouHub** [comment]
Python 3.7 is no longer supported by pytest: https://docs.pytest.org/en/stable/changelog.html#version-compatibility

File: radis/test/io/test_exomol.py [hunk 1]
```diff
@@ -28,8 +28,11 @@ def test_exomol_parsing_functions(verbose=True, *args, **kwargs):
     #  Note : this may change if new databases are added ; test would have to
     # be updated in that case.
     databases, recommended = get_exomol_database_list("CH4", "12C-1H4")
-    assert databases == ["xsec-YT10to10", "YT10to10", "YT34to10", "xsec-MM", "MM"]
-    assert recommended == "YT34to10"
+    assert (
+        databases.sort()
```
Comment (erwanp) [inline radis/test/io/test_exomol.py:32]: top of my head; i think `databases.sort()` returns None  whereas `sorted(databases)` returns the list

**Pull request merged**